src/prime_field.rs

```diff
@@ -469,3 +472,29 @@ where
         self
     }
 }
+
+#[cfg(feature = "serde")]
```

Comment:
I think just having a conditional derive on `FieldElement` should work?
```rust
#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
pub struct FieldElement<T: PrimitiveUint, const M: u64>(T);
```
For once, `serde`'s derivation heuristic would actually work well.